EDAM_dev.owl

```diff
@@ -143,6 +143,7 @@
 
     <owl:AnnotationProperty rdf:about="http://edamontology.org/hasHumanReadableId">
         <rdfs:label>hasHumanReadableId</rdfs:label>
+        <oboInOwl:hasDefinition>An officially maintained and human-readable identifier which is an alternative to the EDAM numerical identifier (URI).</oboInOwl:hasDefinition>
```

Comment:
We shoudl just get rid of this altogether, it's not officially maintained, not persistent, not derefenceable, and most likely (therefore) not used anywhere.

Reply:
Done elsewhere: #799